backend/Tasque.BLL/JWT/JwtIssuerOptions.cs

```diff
@@ -12,7 +12,7 @@ public class JwtIssuerOptions
 
         public string Audience { get; set; }
 
-        public string SecretKey { get; set; }
+        public string Key { get; set; }
 
         public int ValidFor { get; set; }
```

Comment:
Can you fix github actions warnings, please?)

Reply:
Done